src/Core/Constant.js

```diff
@@ -0,0 +1,41 @@
+var Symbol = window.Symbol || function (d) {return 'Symbol('+d+')'}
```

Comment:
`window` reference here breaks node import
`window is not defined`

Reply:
This is what is generated in the bundle –
 [https://github.com/thysultan/dio.js/blob/V8/dist/dio.js](https://github.com/thysultan/dio.js/blob/V8/dist/dio.js). window points to global in commonjs env.

Reply:
dio.js `requires` dio.server.js which seems to include the `window` reference

Reply:
I see, yes your right, that is indeed a problem.